test/test_framework.py

```diff
@@ -1376,6 +1377,24 @@ class StatefulObject(Object):
         #       parent._stored._data.dirty is False?
 
 
+def create_framework(testcase):
+    """Create a Framework object."""
+    framework = Framework(":memory:", charm_dir='non-existant', meta=None, model=None)
+    testcase.addCleanup(framework.close)
+    return framework
```

Comment:
why the duplication with `TestFramework.create_framework`?

Reply:
It's a first step. We need to talk about factory functions. Let's do that on Monday.